aopy/postproc/base.py

```diff
@@ -404,3 +406,138 @@ def _correct_hand_traj(bmi3d_task_data):
     hand_position[bad_pt_mask] = np.nan
 
     return hand_position
+
+def get_bmi3d_mc_input(manual_input, rotation, offset, scale=1):
```

Comment:
hmm yeah good point. i put bmi3d in the function names because they only operate on bmi3d data, not (for example) labgraph data, etc. where the coordinate system isn't messed up haha. but maybe i can drop that from the name and put it in the docstring instead

Reply:
Oh I see. Yeah, since we're dealing with coordinate systems and BMI3D is a category, I think it'd be less confusing to just be get_mc_input, etc.